pallets/schemas/src/lib.rs

```diff
@@ -611,6 +651,280 @@ pub mod pallet {
 				},
 			)
 		}
+
+		/// Creates a new Intent with a name
```

Comment:
should this be only active on testnet and other. Not main-net? if so maybe we need to add in docs

Reply:
Correct. Following the existing pattern for `create_schema`, it's in the README and other places, but not in the Rust docs--but probably a good thing to add.